### DISCUSSION TITLE
PNP search results in clients app- not working

### DISCUSSION THREAD

**Discussion by MaruVC** [microsoft-search/pnp-modern-search#4131] [Q&A]
Hi.
Is it possible to get the PNP search results to open on desktop app when clicking the link?
I tried web=0 (i read somewhere that was the way) but it downloads intested.
I checked both the site and library settings and they are both set to open in clients app, but still, i will either open on the browser or download.

Anything else i can try please?
thank you

**Comment by wobba** [comment]
If you construct url's following the documentation here you can make it work. https://learn.microsoft.com/en-us/office/client-developer/office-uri-schemes And you could probably use `DefaultEncodingURL` property as the URL.
